tests/unit/test_firehose.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from localstack.services.firehose import firehose_api
+
+TEST_STREAM_NAME = 'test_stream'
```

Comment:
Perhaps better use a more descriptive name (or append a random id, we could use `short_uid()` from `util/common.py`) to avoid name clashes with other tests?

Reply:
Agreed. Updated to 'firehose_test_' + short_uid(). Let me know if that's agreeable.